redbot/core/config.py

```diff
@@ -956,7 +956,15 @@ def guild_from_id(self, guild_id: int) -> Group:
         `Group <redbot.core.config.Group>`
             The guild's Group object.
 
+        Raises
+        ------
+        TypeError
+            If the given guild_id parameter is not of type int
         """
+        if type(guild_id) is not int:
+            raise TypeError(
+                f"`guild_id` should be of type `int`, not {guild_id.__class__.__name__}"
```

Comment:
```suggestion
                f"guild_id should be of type int, not {guild_id.__class__.__name__}"
```
The backticks should either be there for all "code" items or for none of them. I personally vote they should not be there.